users/rverst/unicode.h

```diff
@@ -0,0 +1,17 @@
+#ifndef RV_UNICODE_H
+#define RV_UNICODE_H
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

And:
```suggestion
#pragma once
```

Reply:
Sure, sorry, missed that file.